app/Services/Spotify/Endpoints/ArtistClient.php

```
@@ -0,0 +1,183 @@
+
+    /**
+     * Sync DTO back to Spotify API
+     * Artists are read-only in Spotify API.
+     *
+     * @param DtoInterface $dto         The artist DTO
+     * @param string       $resourceId  The Spotify artist ID
+     * @param mixed        $transformer The transformer instance
+     *
+     * @return void
+     */
+    public function syncToSpotify(DtoInterface $dto, string $resourceId, mixed $transformer): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$dto'. (undefined)

(UnusedFormalParameter)